packages/core/src/lib/waku.ts

```diff
@@ -191,7 +165,7 @@ export class WakuNode implements Waku {
   }
 
   async stop(): Promise<void> {
-    this.stopAllKeepAlives();
+    this.connectionManager.stopAllKeepAlives();
```

Comment:
```suggestion
    this.connectionManager.stop();
```

Just expose a `stop` method that handles what needs to be handled.